pyprophet/export.py

```diff
@@ -1,16 +1,21 @@
 import pandas as pd
 import numpy as np
 import sqlite3
```

Comment:
Do we still need sqlite3?

Reply:
Good point probably for export we don't need sqlite3